### PR TITLE
Rename monero-core_rs.ts to monero-core_sr.ts

### PR DESCRIPTION
**Pull request by alexej996**
ISO 639-1 code for Serbian language is "sr".

### CODE REVIEW

**Comment by erciccione** [comment]
Please also edit the `monero-wallet-gui.pro` file:
https://github.com/monero-project/monero-gui/blob/master/monero-wallet-gui.pro#L357

**Comment by erciccione** [comment]
Hey @alexej996 some stuff got merged and now there are some conflicts on this PR. Could you rebase? Let me know if you need help doing so

**Comment by alexej996** [comment]
Well, Win64 build was the only one that failed, but I see it failing for some other pull requests as well.
Anything I can do to help?

File: monero-wallet-gui.pro [hunk 1]
```diff
@@ -327,7 +327,38 @@ macx {
 
 
 # translation stuff
-TRANSLATIONS = $$files($$PWD/translations/monero-core_*.ts)
```
Comment (erciccione) [inline monero-wallet-gui.pro]: Please remove all following strings and leave only `-TRANSLATIONS = $$files($$PWD/translations/monero-core_*.ts)`

**Comment by erciccione** [comment]
Perfect, now can you squash those two commits please?

**Pull request merged**